src/website/views/components/button-group.pug

```diff
@@ -169,3 +169,191 @@ p.-text
         </div>
       </button>
     </div>
+
+h3 Sizes
+p.-text
+  | Button groups supports a full spectrum of sizes: <code>-sm</code>, <code>-md</code>,
+  |  <code>-lg</code>, <code>-xl</code>.
+  | The default size is <code>-md</code>.
+.example.-mb--3
```

Comment:
@davidojedacl can you add tabs for the sizes examples?

Reply:
ok! for sure